app/javascript/hyacinth_v1/components/digital_objects/metadata/MetadataForm.js

```diff
@@ -63,8 +63,12 @@ function MetadataForm(props) {
   );
   const [identifiers, setIdentifiers] = useState(digitalObject.identifiers);
 
+  // ALL CAPS enum value per graphql API
+  const digitalObjectTypeUcase = digitalObjectType.toUpperCase();
```

Comment:
I was expecting the digitalObjectType value here to already be upper case if it came from a graphql response, but there may be something that I missed.